_rules/document-has-main-landmark-b40fd1.md

```diff
@@ -0,0 +1,194 @@
+---
+id: b40fd1
+name: HTML page has a main landmark
+rule_type: atomic
+description: |
+  This rule checks that each page has an element with a semantic role of `main`
+accessibility_requirements:
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _Romance of the Three Kingdoms_ by Luo Guanzhong, translation by Charles Henry Brewitt-Taylor (Tuttle Publishing, 1925, ISBN 9780804834674)
+    - _Three Kingdoms_ by Luo Guanzhong, translation by Moss Roberts (Foreign Language Press, 1976, ISBN 7-119-00590-1)
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectations
+
+Within the test target, there is at least one element for which all the following are true:
```

Comment:
"Within" is ambiguous (same for other rules). What does that mean? Same document? Shadow-included? Can it be in an iframe?

Reply:
Test target are [HTML Web pages](https://act-rules.github.io/glossary/#web-page-html) whose definition contains:
> **Note**: Although web pages as defined here are sets of documents (and do not contain other kind of nodes), one can abusively write that any node is "in a web page" if it is a shadow-including descendant of a document that is part of that web page.

I agree that this is definitely old-ish way of writing stuff and we could improve that definition. But I'd argue that "in" (or "within") an HTML web page is not ambiguous.